packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -82,6 +90,71 @@ const WorkspaceHeader: React.FC = () => {
 
   const [versionAlert, setVersionAlert] = useState(false);
 
+  const [upgradeButtonData, setUpgradeButtonData] = useState<{
+    trialDaysLeft?: number;
+    trialLeftProgress?: number;
+    showUpgradeTrialButton: boolean;
+    showUpgradeDefaultButton: boolean;
+  }>({ showUpgradeTrialButton: false, showUpgradeDefaultButton: true });
+
+  const getCurrentSubscription = useCallback(async () => {
+    if (currentWorkspace) {
+      await stigg.setCustomerId(currentWorkspace.id);
```

Comment:
dont we have this set on our stigg hook ?
if not it should happen on any signin and not on specific check. also it should be part of the context/stigg hook

Reply:
As users will be signed in for days, this check must happen every day they load or move around in the system